cmd/upgrade/executor/upgradetask.go

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright 2019 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package executor
+
+import (
+	"strings"
+
+	"github.com/golang/glog"
+
+	"github.com/openebs/maya/pkg/util"
+	"github.com/spf13/cobra"
+
+	errors "github.com/openebs/maya/pkg/errors/v1alpha1"
+	upgrade100to120 "github.com/openebs/maya/pkg/upgrade/1.0.0-1.1.0/v1alpha1"
+	utask "github.com/openebs/maya/pkg/upgrade/v1alpha2"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	upgradeTaskJobUpgradeCmdHelpText = `
```

Comment:
`upgradeTaskJobUpgradeCmdHelpText` -> `resourceUpgradeCmdHelpText`

Let us rename the variables with `resource` to maintain the readability and avoid confusing with UpgradeTask CR as well as UpgradeTaskJob etc.

Reply:
Done